core/chains/cosmos/config.go

```diff
@@ -36,7 +41,7 @@ func (cs CosmosConfigs) ValidateConfig() (err error) {
 	for i, c := range cs {
 		for j, n := range c.Nodes {
 			if names.IsDupe(n.Name) {
-				err = multierr.Append(err, v2.NewErrDuplicate(fmt.Sprintf("%d.Nodes.%d.Name", i, j), *n.Name))
+				err = multierr.Append(err, v2.NewErrDuplicate(fmt.Sprintf("Cosmos.%d.Nodes.%d.Name", i, j), *n.Name))
```

Comment:
I added chain specifiers to the duplicate error log format since a NOP with multiple chains and multiple duplicates end up with a difficult to parse list of duplicate fields. I think it makes the customer experience better, but it is a break to a pattern that is applied with other config error messages.

Reply:
Aren't these redundant though? These errors are already grouped under `Cosmos`, and when there is just one it should fallback to a direct prefix base case:
![image](https://github.com/smartcontractkit/chainlink/assets/1194128/ec673f4c-354b-4c90-b659-4aa1cce6bfec)

Reply:
Ah, since you aren't going through `v2.Validate()`, you aren't getting the `namedMultiErrorList`, but you can easily do the same. Maybe just export that.